docs/settingup.md

```diff
@@ -1602,7 +1602,7 @@ Explore the hub for apps and fill the artifact map with apps to be used by other
 ### Settings
 
 * `keepcurrent`: Keep the current artifact map and add the results from the `elasticexplore` action. Defaults to `false` (that is, empty the artifact map before adding the results from the `elasticexplore` action), if omitted.
-* `paging`: Go through all app pages in the hub. Defaults to `false` (that is, only include the first 30 apps that the user can see), if omitted.
+* `paging`: Go through all app pages in the hub. Defaults to `false` (that is, only include the first 24 apps that the user can see), if omitted.
```

Comment:
you're not supposed to edit this file directly it's generated. You can find out how generating documentation works here:
https://github.com/qlik-oss/gopherciser/tree/master/generatedocs
Basically you need to edit the `params.json` file then run `go generate`

Reply:
Ok sounds good thanks!

Reply:
I will look at updating the doc in a separate PR

Reply:
OK done!